src/index.js

```diff
@@ -4,6 +4,8 @@ import { identical, getType } from './mitsuketa';
 import err                    from './err';
 import { format }             from './locale';
 import defaultLocale          from './locale/en';
+import WarningBox             from './components/warningbox';
+import LabelContainer         from './components/labelcontainer';
```

Comment:
I already commented about the capitalization. This WILL cause issues for users that are used to normal React conventions and not on a Windows machine.
If we want to keep it this way we will have to very explicitly state that all imports **must** be lowercase. But we also explictly stated how to use this project with `create-react-app` and got lots of issues 🙂